### PR TITLE
The Garnett Silhouette

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Delineate writers' profile pic so that text can wrap around it nicely.

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -1320,3 +1313,52 @@ $quote-mark: 35px;
...
+        width: auto;
+        shape-outside: polygon(0 130px, 0 150px, 180px 150px, 180px 0, 36px 0, 36px 96px);
+
+        @include mq(desktop) {
+            shape-outside: polygon(0 160px, 0 180px, 216px 180px, 216px 0, 36px 0, 24px 121px, 0 128px);
+        }
+    }
+    
+    .byline-img__img {
+        width: 160px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/content-garnett/_types.scss]: Not sure about that though: @superfrank I resized the image to get rid of some negative space around it, but if not necessary I can just remove that rule.

**Comment by regiskuckaertz** [comment]
@superfrank just checked the cutout works with June Udorie's profile pic and it still does 🤗

**Comment by superfrank** [comment]
cc @paperboyo

**Review by superfrank — APPROVED** [review]
👍

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**